docs/migrations/migrations.md

```diff
@@ -0,0 +1,232 @@
+---
+myst:
+  html_meta:
+    "description": "Migrating Plone"
+    "property=og:description": "Migrating Plone"
+    "property=og:title": "Migrating Plone"
+    "keywords": "Versions, Upgrade, Migration, Relaunch, Changes"
+---
+
+(migrations-label)=
+
+# Migrating Plone
+
+## Intro
+
+* What do you want to learn?
+* What are your experiences with migrations so far?
+* What are your next use cases?
+
+
+## What are Migrations?
+
+### Plone Versions
+
+Like most software, Plone changes over time.
+These changes are reflected in different releases of Plone.
+
+The [release-schedule](https://plone.org/download/release-schedule) specifies, which versions are supported and for how long.
+
+```{figure} https://plone.org/download/plonereleaseschedule-2022-09-12.png/@@images/image
+:alt: The Plone release schedule
+```
+
+Maintenance support
+: Bug fixes and small new features are added.
+  Around the end of maintenance support, a final release will be done.
+
+Security support
+: Plone security hotfixes will be made available for this series.
+  For more information, see the [security update policy](https://plone.org/security/update-policy).
+
+Currently, Plone 6.0.x is under maintenance support and Plone 5.2.x is under security support.
+That means new features are being developed for Plone 6.1.x, and bugfixes are being developed for Plone 6.0.x.
+
+Plone intends to provide stable releases (for example, 5.2.8 is a stable release of the 5.2.x series).
+Only bugfixes and changes that extend or improve an existing feature—as long as they don't break anything—make it into a bugfix release.
+
+
+### What's in a Plone version?
+
+In fact, it is even more complicated, because Plone consists of about 250 separate Python packages, all of which have their own versions and history.
```

Comment:
+ many of these python packages are developed and released 'inside' the Plone and Zope projects, but Plone also depends on more generic Python packages developed by third parties